NitroxPatcher/Patches/SubNameInput_OnNameChange_Patch.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Harmony;
+using NitroxClient.GameLogic.Helper;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.Helper;
+using System;
+using System.Reflection;
+
+namespace NitroxPatcher.Patches
+{
+    public class SubNameInput_OnNameChange_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(SubNameInput);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("OnNameChange", BindingFlags.Public | BindingFlags.Instance);
+
+        public static void Postfix(SubNameInput __instance)
+        {
+            SubName subname = (SubName)__instance.ReflectionGet("target");
```

Comment:
It looks like OnNameChange() has a null check for target.  We should probably put in a similar check here.